client/tailwind.config.js

```diff
@@ -5,6 +5,29 @@ module.exports = {
     extend: {
       colors: {
         "sc-blue": "#0077c0",
+        "sc-dark-blue": "#00629e",
+      },
+      keyframes: {
+        xTranslation: {
+          "0%": {
+            width: "3.375rem",
+          },
+          "100%": {
+            width: "100%",
+          },
+        },
+        yTranslation: {
+          "0%": {
+            height: "3.375rem",
+          },
+          "100%": {
+            height: "50vh",
+          },
+        },
+      },
+      animation: {
+        xTranslation: "xTranslation 7.57s linear infinite alternate",
```

Comment:
As Youssef suggested, it maybe better to name these class names more specifically.

Reply:
So in tailwind.config, the object has to be called animation and under it you can add as many key-value pairs as you want as we add more animations. So I can't change, the word "animation" in this piece of code.

Or do you mean my naming of xTranslation and yTranslation? If so I can change those.